src/components/Referrals/ConfirmInviteUser.vue

```diff
@@ -0,0 +1,97 @@
+<template>
```

Comment:
1) If this is Polkaswap logo, should the background be white?
2) In figma the icon is checkmark, not chevron

![invitation](https://user-images.githubusercontent.com/53777036/152290827-87e7bdbc-e449-4944-9909-e03fc6f1b931.jpg)

Reply:
1. I don't think that that we should use white color here (because we have have another pink color as well, so this icon is not the same with logo icon).
2. Thank you, fixed!